tutorials/05_style.md

```diff
@@ -12,7 +12,7 @@ but it is also possible to use others such as Default and Universal. This tutori
 focuses on customizing the Material style.
 
 The default style is hardcoded into the
-[qtquickcontrols2.conf](https://github.com/ignitionrobotics/ign-gui/blob/master/include/ignition/gui/qtquickcontrols2.conf)
+[qtquickcontrols2.conf](https://github.com/ignitionrobotics/ign-gui/blob/ign-gui3/include/ignition/gui/qtquickcontrols2.conf)
```

Comment:
Should this be changed to `ign-gui4`?

```suggestion
[qtquickcontrols2.conf](https://github.com/ignitionrobotics/ign-gui/blob/ign-gui4/include/ignition/gui/qtquickcontrols2.conf)
```

Reply:
Yes, please! You can add it as a new commit. Just add `(#189)` to the end of the commit to it has a link back to this PR. Thanks!